packages/design-system/src/utils/pointerEventsCss.ts

```diff
@@ -19,7 +19,7 @@
  */
 import { css } from 'styled-components';
 
-const pointerEventsCss = css`
+const pointerEventsCss = css<{ pointerEvents?: string }>`
   ${({ pointerEvents }) => {
     if (pointerEvents && typeof pointerEvents === 'string') {
```

Comment:
Does it still need this check for `string`?

Reply:
Hmm, why do we even have this util? It's for adding a single line of CSS with an util which is almost identical to the CSS you would write anyway? We actually even have this stupid util in two different packages! I mean, why? I'll kill both, they're only ever used once and twice respectively!!! It's too stupid to keep as an util!

Reply:
Removed in 8a04a9d497.